fsd_config/form_jsons/dpi_r2/dataset-information-dpi.json

```diff
@@ -476,5 +476,6 @@
     ],
     "version": 2,
     "skipSummary": false,
-    "markAsComplete": false
+    "markAsComplete": false,
+    "name":"Digital Planning Improvement Fund"
```

Comment:
Shouldn't it say "”Apply for funding to begin your digital planning improvement journey ”?

This is how we display it for CYP and COF

![image](https://github.com/communitiesuk/digital-form-builder/assets/36962596/51c0723d-89a9-4c8d-a5f1-5fa69542c74b)

![image](https://github.com/communitiesuk/digital-form-builder/assets/36962596/8410462d-5236-4524-817c-d1d3c01eb92d)

Reply:
Changed this and updated the image